appeals/api/src/server/endpoints/constants.js

```diff
@@ -67,6 +67,9 @@ export const ERROR_VALID_VALIDATION_OUTCOME_REASONS_REQUIRED =
 	'Validation outcome reasons are required when validationOutcome is Incomplete or Invalid';
 export const ERROR_LPA_QUESTIONNAIRE_VALID_VALIDATION_OUTCOME_REASONS_REQUIRED =
 	'Validation outcome reasons are required when validationOutcome is Incomplete';
+export const ERROR_APPEAL_DUE_DATE_MUST_BE_IN_FUTURE = 'Appeal due date must be in the future';
+export const ERROR_LPA_QUESTIONNAIRE_DUE_DATE_MUST_BE_IN_FUTURE =
+	'LPA questionnaire due date must be in the future';
```

Comment:
The API returns an object with the field name as a parameter so it's clear which field the error relates to, for example

```
{
  "errors": {
    "appealDueDate": "Must be in the future"
  }
}
```

so perhaps it would be better to have a single, more generic error message that could be used for both, then if we need to use for a different field in the future the same error could be reused again, something like

`ERROR_MUST_BE_IN_FUTURE = 'Must be in the future'`

Reply:
Good idea, thanks! Updated.